example/src/main/scala/xs4s/example/package.scala

```diff
@@ -27,6 +28,16 @@ package object example {
       .map(_.filter(_._2 > 1))
       .map(_.toList.sortBy { case (keyword, count) => -count })
 
+  def countTopItemsZIO[R, I]: ZStream[R, Throwable, I] => ZStream[R, Throwable, List[(I, Int)]] = { s =>
+    val res = s.fold(Map.empty[I, Int].withDefaultValue(0)){ (acc: Map[I, Int], v: I) =>
+      acc.updated(v, acc(v)+1)
+    }
+      .map(_.filter(_._2 > 1))
+      .map(_.toList.sortBy { case (keyword, count) => -count })
+
+    ZStream.fromEffect(res)
```

Comment:
I did verify there is no current way to make a stream => fold (ZIO) => back into a stream without storing the value and using the fromEffect